packages/cli/src/commands/setup/auth/templates/auth.js.template

```diff
@@ -13,42 +12,71 @@ import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/api'
  *
  * @see https://github.com/redwoodjs/redwood/tree/main/packages/auth for examples
  */
-export const getCurrentUser = async (decoded, { _token, _type }, { _event, _context }) => {
+export const getCurrentUser = async (
+  decoded,
+  { _token, _type },
+  { _event, _context }
+) => {
   return { ...decoded, roles: parseJWT({ decoded }).roles }
 }
 
+/**
+ * The user is authenticated if there is a currentUser in the context
+ *
+ * @returns {boolean} - If the currentUser is authenticated
+ */
+export const isAuthenticated = () => {
+  return !!context.currentUser
+}
+
+/**
+ * Checks if the currentUser is authenticated (and assigned one of the given roles)
+ *
+ * @param {string= | string[]=} roles - A single role or list of roles to check if the user belongs to
```

Comment:
```suggestion
 * @param {string | string[]} roles - A single role or list of roles to check if the user belongs to
```

Reply:
I can't make up my mind about this one

The code clearly handles that you don't pass a role in. But on the other hand it feels to weird to call `hasRole()`. From just looking at that, without reading the implementation, it's not very easy to know what it would do. Would it check if the user has any role? Any role at all? Or would it check if the user has the role `undefined`?